multisig/proposals/ethereum/vip-073/index.ts

```diff
@@ -0,0 +1,214 @@
+      return {
+        target: converter,
+        signature: "transferOwnership(address)",
+        params: [ethereum.NORMAL_TIMELOCK],
+      };
+    }),
+    {
+      target: PRIME,
+      signature: "transferOwnership(address)",
+      params: [ethereum.NORMAL_TIMELOCK],
+    },
+    {
+      target: PLP,
+      signature: "transferOwnership(address)",
+      params: [ethereum.NORMAL_TIMELOCK],
+    },
+    ...REWARD_DISTRIBUTORS.map(rewardDistributor => {
+      return {
+        target: rewardDistributor,
+        signature: "transferOwnership(address)",
+        params: [ethereum.NORMAL_TIMELOCK],
+      };
+    }),
+    {
+      target: PSR,
+      signature: "transferOwnership(address)",
+      params: [ethereum.NORMAL_TIMELOCK],
+    },
+    {
+      target: COMPTROLLER_BEACON,
+      signature: "transferOwnership(address)",
+      params: [ethereum.NORMAL_TIMELOCK],
+    },
+    {
+      target: VTOKEN_BEACON,
+      signature: "transferOwnership(address)",
+      params: [ethereum.NORMAL_TIMELOCK],
+    },
+    {
+      target: POOL_REGISTRY,
+      signature: "transferOwnership(address)",
+      params: [ethereum.NORMAL_TIMELOCK],
+    },
+    ...COMPTROLLERS.map(comptroller => {
+      return {
+        target: comptroller,
+        signature: "transferOwnership(address)",
+        params: [ethereum.NORMAL_TIMELOCK],
+      };
+    }),
+    ...VTOKENS.map(comptroller => {
+      return {
+        target: comptroller,
+        signature: "transferOwnership(address)",
+        params: [ethereum.NORMAL_TIMELOCK],
+      };
+    }),
+    {
+      target: ethereum.XVS_VAULT_PROXY,
+      signature: "_setPendingAdmin(address)",
+      params: [ethereum.NORMAL_TIMELOCK],
+    },
+    {
+      target: XVS_STORE,
+      signature: "setPendingAdmin(address)",
+      params: [ethereum.NORMAL_TIMELOCK],
+    },
+    {
+      target: XVS_BRIDGE_ADMIN_PROXY,
+      signature: "transferOwnership(address)",
+      params: [ethereum.NORMAL_TIMELOCK],
+    },
+    {
+      target: XVS,
+      signature: "transferOwnership(address)",
+      params: [ethereum.NORMAL_TIMELOCK],
+    },
+    {
+      target: ethereum.RESILIENT_ORACLE,
+      signature: "transferOwnership(address)",
+      params: [ethereum.NORMAL_TIMELOCK],
+    },
+    {
+      target: ethereum.CHAINLINK_ORACLE,
+      signature: "transferOwnership(address)",
+      params: [ethereum.NORMAL_TIMELOCK],
+    },
+    {
+      target: ethereum.REDSTONE_ORACLE,
+      signature: "transferOwnership(address)",
+      params: [ethereum.NORMAL_TIMELOCK],
+    },
+    {
+      target: BOUND_VALIDATOR,
+      signature: "transferOwnership(address)",
+      params: [ethereum.NORMAL_TIMELOCK],
+    },
+    {
+      target: SFrxETHOracle,
+      signature: "transferOwnership(address)",
+      params: [ethereum.NORMAL_TIMELOCK],
+    },
+    {
+      target: ethereum.VTREASURY,
+      signature: "transferOwnership(address)",
+      params: [ethereum.NORMAL_TIMELOCK],
+    },
```

Comment:
The VTreasury on Ethereum was transferred to the Normal Timelock. I would update the block number of every simulation to confirm this and maybe identify other issues